collect_app/src/main/java/org/odk/collect/android/configure/SettingsImporter.kt

```diff
@@ -50,10 +52,17 @@ class SettingsImporter(
             } else {
                 JSONObject()
             }
+
+            val connectionIdentifier = if (generalSettings.getString(GeneralKeys.KEY_PROTOCOL).equals(Collect.getInstance().getString(R.string.protocol_google_sheets))) {
```

Comment:
We have `projectDetailsShouldBeImportedIfIncludedInJson` test but now as we have the second case maybe it would be good to add a test for it as well?